src/templates/aws/advanced.ts

```diff
@@ -191,21 +329,100 @@ export default class Advanced {
 
     const bastionModuleContent = dedent`
     module "bastion" {
-      source = "../modules/bastion"
-    
+      source = "./modules/bastion"
+
       subnet_ids                  = module.vpc.public_subnet_ids
       instance_security_group_ids = module.security_group.bastion_security_group_ids
-    
+
       namespace     = var.namespace
       image_id      = var.bastion_image_id
       instance_type = var.bastion_instance_type
-    
+
       min_instance_count     = var.bastion_min_instance_count
       max_instance_count     = var.bastion_max_instance_count
       instance_desired_count = var.bastion_instance_desired_count
     }
     `
 
-    injectToFile('main.tf', bastionModuleContent, this.options, {insertAfter: '# Bastion instance'})
+    injectToFile('main.tf', bastionModuleContent, this.options, {
+      insertAfter: '# Bastion instance',
+    })
+  }
+
+  private applyEcs(): void {
+    copyDir('aws/modules/ecs', 'modules/ecs', this.options)
+
+    const bastionVariablesContent = dedent`
+    variable "ecr_repo_name" {
+      description = "ECR repo name"
+      type = string
+    }
+
+    variable "ecr_tag" {
+      description = "ECR tag to deploy"
+      type     = string
+    }
+
+    variable "ecs" {
+      description = "ECS input variables"
+      type = object({
+        task_cpu                           = number
+        task_memory                        = number
+        task_desired_count                 = number
+        task_container_memory              = number
+        deployment_maximum_percent         = number
+        deployment_minimum_healthy_percent = number
+      })
+    }
+
+    variable "deeplink_email_verification" {
+      description = "Email Verification Deeplink"
+      type        = string
+    }
+
+    variable "aws_sns_sender_id" {
+      description = "AWS SNS sender ID"
+      type        = string
+    }
+
+    variable "aws_access_key_id" {
+      description = "AWS access key ID"
+      type        = string
+    }
+
+    variable "aws_secret_access_key" {
+      description = "AWS secret access key"
+      type        = string
+    }\n\n`
```

Comment:
I think we don't need these variables?

Reply:
Yes, while copy/paste, forgot to delete 😄